### PR TITLE
Block access to lucene and other supporting files

### PR DESCRIPTION
**Pull request by camfindlay**

### CODE REVIEW

**Comment by camfindlay** [comment]
@assertchris can I get a thumbs up on this ;)

File: .htaccess_live [hunk 1]
```diff
@@ -1,357 +0,0 @@
...
-	RewriteRule ^doku.php$                http://doc.silverstripe.org/%2?%3 [R=301,L]
-	
-	# Redirect legacy URLs (assumes we're not redirecting other assets, for performance reasons)
-	# Note: Just works on top-level domains, not if the webroot is in a subfolder
-	RewriteCond %{REQUEST_URI} ^(.*)$
-	RewriteCond %{REQUEST_FILENAME} !-f
-	RewriteCond %{REQUEST_URI} !\.(css|gif|ico|jpg|js|png|swf|txt)$
-	#RewriteRule ^execution-pipeline$ http://doc.silverstripe.org/framework/en/reference/ [R=301,L]
-		
-	RewriteRule ^arraydata$ http://api.silverstripe.org/search/lookup/?q=ArrayData&module=sapphire [R=301,L]
```
Comment (assertchris) [inline .htaccess_live]: What happens to all of these rewrites?

Comment (camfindlay) [inline .htaccess_live]: This is in a .htaccess_live file which we never use... tidying up.

Comment (assertchris) [inline .htaccess_live]: Should those rules be put in place?

Comment (camfindlay) [inline .htaccess_live]: Last touched 3 years ago... and not actually used on the site.
screen shot 2016-02-11 at 12 44 36 pm

**Comment by assertchris** [comment]
:+1:

**Pull request merged**